test/compile/mark/bar.test.ts

```diff
@@ -81,7 +86,12 @@ describe('Mark: Bar', function() {
     const model = parseUnitModel({
         "mark": "bar",
         "encoding": {"x": {"type": "quantitative", "field": 'US_Gross', "aggregate": 'sum'}},
-        "data": {"url": 'data/movies.json'}
+        "data": {"url": 'data/movies.json'},
```

Comment:
You're fixing default behavior in `config.ts`, but here modify the test to no longer test the default behavior.  The correct fix is to fix the asserts below.